src/main/context.ts

```diff
@@ -256,50 +269,31 @@ async function getEventDefaults(appName: string | undefined): Promise<Event> {
         name: 'Electron',
         version: process.versions.electron,
       },
+      electron: {
+        crashed_process: 'browser',
+      },
     },
     environment: process.defaultApp ? 'development' : 'production',
-    extra: { crashed_process: 'browser' },
-    release: `${name.replace(/\W/g, '-')}@${app.getVersion()}`,
+    release: release || `${name.replace(/\W/g, '-')}@${app.getVersion()}`,
     user: { ip_address: '{{auto}}' },
+    tags: { event_type: 'javascript' },
```

Comment:
I noticed that @jan-auer had added `tags.event_type` [here](https://github.com/getsentry/sentry-electron/commit/20d96580593d98ab062f5e827f680025843e3b62) so I was reluctant to change it as users will likely be using it to filter events.

I will look at the different options for `event.environment`!

Reply:
Initially after reading your comment I became confused by `event.environment`. I assumed this would clash with `environment` set by customers (ie. dev, staging, prod etc). Now I'm looking closer I see this is actually keys for the tags.

What would I set `origin` and `environment` tagd to for Electron JavaScript errors and native crashes? 

Something like:

JavaScript errors:
`event.origin`: `electron`
`event.environment`: `javasript`

Native crashes:
`event.origin`: `electron`
`event.environment`: `native`

Reply:
Yeah I think what you've outlined with javascript/native makes the most sense.

Reply:
What about `event.origin: javascript` to match what's done on React Native, Cordova, Capacitor?
It basically means this came from JS code. @jennmueng can correct me here.
If it was a native event then it would be `event.origin:native`.

Unless I get this wrong, we really need to get this on the develop docs. Maybe @marandaneto who did similar thing in Flutter can chime in

Reply:
Yes we use `event.origin: javascript` on RN

Reply:
🤔 If `event.origin` gives the language that the event has come from, what is `event.environment` supposed to signify?